images/nginx/build.sh

```diff
@@ -204,6 +204,14 @@ export MAKEFLAGS=-j${CORES}
 export CTEST_BUILD_FLAGS=${MAKEFLAGS}
 export HUNTER_JOBS_NUMBER=${CORES}
 
+# Installing luarocks packages
+if [[ ${ARCH} == "x86_64" ]]; then
+  luarocks install lrexlib-pcre 2.7.2-1
+fi
+
+luarocks install lua-cjson
+luarocks install luasocket
```

Comment:
this should not be necessary for that PR, we already have `ngx.socket` we can use

Reply:
removed